utils/cq/daos_pylint.py

```diff
@@ -17,8 +17,21 @@
     from pylint.reporters.collecting_reporter import CollectingReporter
     from pylint.constants import full_version
 except ImportError:
-    print('install pylint to enable this check')
-    sys.exit(0)
+
+    if os.path.exists('venv'):
+        sys.path.append(os.path.join('venv', 'lib',
+                                     f'python{sys.version_info.major}.{sys.version_info.minor}',
+                                     'site-packages'))
+        try:
+            from pylint.lint import Run
+            from pylint.reporters.collecting_reporter import CollectingReporter
+            from pylint.constants import full_version
+        except ImportError:
+            print('detected venv unusable, install pylint to enable this check')
+            sys.exit(0)
+    else:
+        print('install pylint to enable this check')
+        sys.exit(0)
```

Comment:
There are no other paths
```suggestion
    print('install pylint to enable this check')
    sys.exit(0)
```

Reply:
I don't think that would work as if the import works on L28 then it would catch this and we don't want it to.